examples/sqlalchemy/orm.py

```diff
@@ -24,11 +25,12 @@ def dump(self):
         return {k: v for k, v in vars(self).items() if not k.startswith("_")}
 
 
-def init_db(uri):
-    engine = create_engine(uri, convert_unicode=True)
-    db_session = scoped_session(
-        sessionmaker(autocommit=False, autoflush=False, bind=engine)
+def init_db():
+    # https://stackoverflow.com/questions/6519546/scoped-sessionsessionmaker-or-plain-sessionmaker-in-sqlalchemy
+    engine = create_engine(
```

Comment:
Thank you for adding these options to the code. Maybe it is also a good idea to add a comment mentioning that `check_same_thread` and `StaticPool` can be helpful for unit testing of in-memory sqlite databases and should not be used in production.

Reply:
Will add the comment.